admin_guide/garbage_collection.adoc

```diff
@@ -70,6 +70,29 @@ kubeletArguments:
 ----
 ====
 
+=== Detection of containers for deletion
+
+At the moment only docker and rkt are supported. Rkt has its own GC. The following is only docker related.
+
+Each spin of the garbage collector loop goes through the following steps:
+
+1. Retrieve a list of available containers
+2. Filter out all containers that are running or are not alive longer
+   than `*minimum-container-ttl-duration*` parameter
+3. Determine pod membership of each remaining container
+4. Remove all unidentified containers (with no pod membership)
+5. For each pod that contains more containers than
+   `*maximum-dead-containers-per-container*`, sort the containers by creation time
```

Comment:
that's what it is now, maybe a more appropriate name would be ```maximum-dead-containers-per-pod-container```. You're welcome to submit a PR to kube :)